demos/rag_agentic/notebooks/Level0_getting_started_with_Llama_Stack.ipynb

```diff
@@ -21,7 +21,7 @@
    "source": [
     "## Installing Dependencies\n",
     "\n",
-    "This code requires `llama-stack` and the `llama-stack-client`, both at version `0.2.2`. Lets begin by installing them:"
+    "This code requires `llama-stack` and the `llama-stack-client`, both at version `0.2.4`. Lets begin by installing them:"
```

Comment:
Can we remove this line? It will be tough to maintain each time we update the llama stack version.